Basic/SourceFilter/NiceSlice.pm

```diff
@@ -5,9 +5,9 @@ BEGIN {
       'Module::Compile'     => 'PDL/NiceSlice/ModuleCompile.pm',
    );  # to validate names
 
-   $PDL::NiceSlice::engine = $engine_ok{'Filter::Simple'};  # default engine type
+   ## $PDL::NiceSlice::engine = $engine_ok{'Filter::Simple'};  # default engine type
    ## TODO: Add configuration argument to perldl.conf
-   ## $PDL::NiceSlice::engine = $engine_ok{'Filter::Util::Call'};  # default engine type
+   $PDL::NiceSlice::engine = $engine_ok{'Filter::Util::Call'};  # default engine type
```

Comment:
It's a pity this is necessary for now, but evidently it is. We should probably add a test exercising the case in #25 / https://sourceforge.net/p/pdl/bugs/238/